generate.sh

```diff
@@ -29,9 +29,8 @@ function generateDocs {
     sleep 3
     git submodule update --recursive --force
     php ../app/composer.phar self-update
-    php ../app/composer.phar install || true
+    php ../app/composer.phar install
     cd ..
-    sleep 4
```

Comment:
We might be able to keep the sleep here (for whatever reason it was added there).
I think the problem was that it was trying to run the generator before the install finished, so removing `|| true` above should do the trick on its own.